bin/dev-restore-db

```diff
@@ -13,15 +13,23 @@ fi
 # set TRUNCATE_SQL to heredoc contents
 TRUNCATE_SQL=$(
   cat <<EOF
-TRUNCATE TABLE files;
-TRUNCATE TABLE applications CASCADE;
-TRUNCATE TABLE applicants CASCADE;
 TRUNCATE TABLE accounts CASCADE;
```

Comment:
(blocking) Since the dump all script exports everything  you'll need to either not export the evolutions tables are truncate them on import.

If would say don't export them and then you don't have to truncate. This way you can restore the database without stopping the application. If someone wants to have no evolutions they can delete the db volume and run fresh.